v2/api/batch/v1alpha1api20210101/batch_account_types_gen.go

```diff
@@ -787,6 +798,83 @@ func (account *BatchAccount_Spec) AssignProperties_To_BatchAccount_Spec(destinat
 	return nil
 }
 
+// Initialize_From_BatchAccount_STATUS populates our BatchAccount_Spec from the provided source BatchAccount_STATUS
+func (account *BatchAccount_Spec) Initialize_From_BatchAccount_STATUS(source *BatchAccount_STATUS) error {
+
+	// AutoStorage
+	if source.AutoStorage != nil {
+		var autoStorage AutoStorageBaseProperties
+		err := autoStorage.Initialize_From_AutoStorageProperties_STATUS(source.AutoStorage)
+		if err != nil {
+			return errors.Wrap(err, "calling Initialize_From_AutoStorageProperties_STATUS() to populate field AutoStorage")
+		}
+		account.AutoStorage = &autoStorage
+	} else {
+		account.AutoStorage = nil
```

Comment:
It's more a side effect thing, given that I reused `PropertyAssignmentFunction` to generate this code. I don't think it hurts to be thorough.

Reply:
Even so, since technically this works as `Assign`, is there a reason not to call it that? It still makes sense in the initialize case anyway, right?

Reply:
I deliberately used a different name to avoid confusion - I figure if future devs see there are three `Assign*()` methods they'll assume they're related, and then try to work out why.